packages/components/src/components/hds/advanced-table/index.ts

```diff
@@ -87,11 +87,11 @@ const getScrollIndicatorDimensions = (
 
   return {
     bottom: `${horizontalScrollBarHeight}px`,
-    height: `${scrollWrapper.clientHeight - horizontalScrollBarHeight}px`,
+    height: `${scrollWrapper.offsetHeight - horizontalScrollBarHeight}px`,
     left: `${leftOffset}px`,
     right: `${verticalScrollBarWidth}px`,
     top: hasStickyHeader ? `${theadElement.offsetHeight}px` : '0px',
-    width: `${scrollWrapper.clientWidth - verticalScrollBarWidth}px`,
+    width: `${scrollWrapper.offsetHeight - verticalScrollBarWidth}px`,
```

Comment:
```suggestion
    width: `${scrollWrapper.offsetWidth - verticalScrollBarWidth}px`,
```
[Issue] Mixed up the width and height here. It's just causing some issues with the bottom and top bar widths right now.